src/capi.rs

```diff
@@ -356,6 +379,42 @@ pub unsafe extern fn rav1e_config_default() -> *mut Config {
   Box::into_raw(c)
 }
 
+/// Setup a single pass rate control
+///
```

Comment:
This is extremely confusing.

Does this mean 1-pass ABR *only*? As in, if I call this function, I can never do a second pass? If so, this is a bad API, and doesn't account for future uses of e.g. CRF first pass, followed by a second pass.

Or does it mean the first of two passes?

What ar its requirements and restrictions on what must be set?

Reply:
It simply reset to our default, single pass rate control with no packet data to emit or consume.

Reply:
That is too rigid. It doesn't allow for arbitrary first passes, like the CRF first pass I descibed above, as far as I can tell, which is important to support.

Reply:
This function has to be used ONLY if you happen to have a Configuration set up for a second pass encoding and you want to reuse it to spawn a single pass encoding context.

Another way to support this can be:
``` c
rav1e_config_set_rc_summary(rac, NULL, 0);
```
But feels fairly uglier.

Configuration is just a stateless factory of Context: Changing the configuration has no impact to the Contexts spawned from it in the past.

Reply:
If you skip this function and instead just use rav1e_config_set_rc_summary(rac, NULL, 0) - do you actually need to call that? Or is the default configuration, e.g. I set up a final pass, but don't provide a rc summary, then it's assumed to be single pass?

Reply:
The latter. By default it is single pass, you do not have to call any function to configure it if you did not set a summary before.

Reply:
As discussed the function got removed, moving the reset capability to 
``` c
rav1e_config_set_rc_summary(rac, NULL, 0);
```